app/Http/Controllers/PDFController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Barryvdh\DomPDF\Facade\Pdf;
+
+class PDFController extends Controller
+{
+
+    public function pdf_generator_get(Request $request)
+    {
+        $users = User::get();
+        $userNames = $users->pluck('name');
+        $data = [
+            'title' => 'PDF',
```

Comment:
Please make this more friendly. You can try "PDF[name][date]"

Reply:
I've removed it